src/ContainerLogs.jsx

```diff
@@ -119,9 +119,8 @@ class ContainerLogs extends React.Component {
                            this.onStreamMessage, true)
                 .then(this.onStreamClose)
                 .catch(e => {
-                    const error = JSON.parse(new TextDecoder().decode(e.message));
                     this.setState({
-                        errorMessage: error.message,
+                        errorMessage: e.message ?? e.toString(),
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2214-e5ac0ddf-20250801-075515-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)

Reply:
It's a "should not happen", podman API should always deliver errors with `message`.